### ISSUE TITLE
SegmentationFault after 2022/10/21 CS:GO update

### ISSUE THREAD

**Issue by Frag1337** [Source-Python-Dev-Team/Source.Python#457]
Most likely invalid signatures as sourcemod just released an gamedata update.

**Comment by Ayuto** [comment]
Thanks! There is more than just signature update.
https://github.com/alliedmodders/hl2sdk/pull/110

**Comment by CookStar** [comment]
Is this still a reproducible issue?

**Comment by Frag1337** [comment]
> Is this still a reproducible issue?

Yes, as this happens on initial plugin load. I sadly don't know what import causes this.

**Comment by CookStar** [comment]
If you are running [Version: 717](http://downloads.sourcepython.com/) and if it crashes, please raise any plugins you know of and I'll take a look at them.

**Comment by CookStar** [comment]
I just realized that Version: 717 was before the hl2sdk commits were merged.

**Comment by Ayuto** [comment]
Sorry for the late reply. A new build should be available in ~45 minutes.
